activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -49,6 +56,12 @@ def test_raises_an_error_when_called_with_protected_environment
       ActiveRecord::Base.protected_environments = [current_env]
       assert_raise(ActiveRecord::ProtectedEnvironmentError) do
         ActiveRecord::Tasks::DatabaseTasks.check_protected_environments!
+        ActiveRecord::Base.protected_environments = [current_env]
+
+        assert_raise(ActiveRecord::ProtectedEnvironmentError) do
+          ActiveRecord::Tasks::DatabaseTasks.check_protected_environments!
+        end
+
```

Comment:
Is there reason adding this? Why do we need to assert it twice?

Reply:
No reason whatsoever. It's a mix-up. Removing.